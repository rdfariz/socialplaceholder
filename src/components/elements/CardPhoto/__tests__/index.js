import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Card from '../';

describe('src/components/elements/CardPhoto', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(
      <Card
        title="title"
        thumbnail="https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"
        url="https://google.com"
      />
    );
    expect(tree).toMatchSnapshot();
  });
});
