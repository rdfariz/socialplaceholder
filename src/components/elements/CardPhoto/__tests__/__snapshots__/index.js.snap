// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/CardPhoto render 1`] = `
<div
  className=" flex"
>
  <div
    className="w-full bg-white overflow-hidden rounded-lg border cursor-pointer hover:shadow-md"
    onClick={[Function]}
  >
    <img
      alt="title"
      className="w-full"
      src="https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"
    />
    <div
      className="p-3 md:p-5 "
    >
      <h2
        className="text-gray-800 text-lg font-semibold"
      >
        title
      </h2>
    </div>
  </div>
</div>
`;
